app/app.js

```diff
@@ -430,6 +450,21 @@ class MaterialColors {
       this._lastCopiedColor = clipboardText;
     }
   }
+
+  _loadConfig() {
+    const configFilePath = path.join(this._getHomeDirectory(), CONFIG_FILENAME);
+
+    try {
+      return yaml.safeLoad(fs.readFileSync(configFilePath, 'utf8')) || {};
+    } catch(e) {
+      // return empty config if config file doesn't exists.
+      return {};
+    }
+  }
+
+  _getHomeDirectory() {
```

Comment:
somehow, electron.app.getPath() isn't working :/ 

Left this untouched for now. Can you please clean this up?